src/test/kotlin/io/bazel/worker/JavaPersistentWorkerTest.kt

```diff
@@ -73,8 +67,15 @@ class PersistentWorkerTest {
       }
       requests.forEach { writeStdIn(it) }
       closeStdIn()
-      return@inProcess generateSequence { readStdOut() }
-    }.associateBy { wr -> wr.requestId }
+      waitForStdOut()
+      return@inProcess generateSequence {
+        readStdOut().apply {
+          println("sequence $this")
+        }
+      }
+    }.associateBy { wr ->
```

Comment:
wr? I know you didn't name it here, but can you drive-by rename with something more clear?